packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -2457,7 +2760,7 @@ void main() {
 
       // Tap after releasing the overscroll to trigger secondary inner ballistic
       // scroll activity with 0 velocity.
-      await tester.tap(find.text('Item 49'), warnIfMissed: false);
+      await tester.tap(find.text('Item 49'));
```

Comment:
I believe the `warnIfMissed` was here due to the fact that the scrollable absorbed all inputs. So the author of the test just put in `warnIfMissed: false` to get rid of the error they were getting. Now that overscrolled scrollable correctly allow input on children the child is no longer missed and this can be removed. I believe keeping it there is not necessary as no warning will be produced now.